portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
```

Comment:
I would suggest putting this file into a separate pull request, since it seems like a separate feature